pypuf/simulation/arbiter_based/ltfarray.py

```diff
@@ -557,5 +558,44 @@ def ltf_eval(self, inputs):
         Random numbers are drawn from the PRNG instance generated when
         initializing the NoisyLTFArray.
         """
-        noise = self.random.normal(loc=0, scale=self.sigma_noise, size=(1, self.k))
-        return super().ltf_eval(inputs) + noise
+        evaled_inputs = super().ltf_eval(inputs)
+        noise = self.random.normal(loc=0, scale=self.sigma_noise, size=shape(evaled_inputs)[0:2])
```

Comment:
`size=(len(inputs), self.k)` is easier to understand than `shape(evaled_inputs)[0:2]`, but for correct implementation of `ltf_eval` equivalent. (Also, why use `[0:2]` on a length 2 array?)

This change should be in a seperate bugfix commit.